dashboard/v1.1/src/layouts/BaseLayout.tsx

```diff
@@ -120,15 +143,19 @@ export default function BaseLayout(props: any): ReactElement {
             >
               Bench Routes
             </Typography>
-            <Tooltip title="Dark Mode">
-              <Switch
-                checked={props.darkMode}
-                onChange={handleToggleDarkMode}
-                color="default"
-                name="checkedB"
-                inputProps={{ 'aria-label': 'primary checkbox' }}
-              />
-            </Tooltip>
+            <Switch
+              checked={props.darkMode}
+              onChange={handleToggleDarkMode}
+              offColor="#145D97"
+              onColor="#303030"
+              name="checkedB"
```

Comment:
Let's remove these prop (i do not think it is necessary)

Reply:
We can remove the name prop but the two color props are required because the default colors of the component does not match our theme.